arviz/tests/test_plots.py

```diff
@@ -55,19 +55,7 @@ def clean_plots(request, save_figs):
 
     def fin():
         if save_figs is not None:
-
-            # Retry save three times
-            for i in range(3):
-                try:
-                    plt.savefig("{0}.png".format(os.path.join(save_figs, request.node.name)))
-
-                except Exception as err:  # pylint: disable=broad-except
-                    if i == 2:
-                        raise err
-                    time.sleep(0.250)
-                else:
-                    break
-
```

Comment:
I don't think this retry loop was really needed for anything. I remember we had a discussion concerning this, but still, I feel like this is not needed. The error was something else.

is not needed == doesn't actually fix any problems, because even if the user has the image open, closing them in 0.75 s is just overengineering on our part.

Reply:
If you want to do it in another PR you can. I can merge it with master.